libsolidity/lsp/LanguageServer.cpp

```diff
@@ -186,6 +186,16 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 			lspAssert(false, ErrorCode::InvalidParams, "Invalid file load strategy: " + text);
 	}
 
+	if (auto traceLogFileNode = _settings["trace-log-file"])
+	{
+		if (traceLogFileNode.isString())
+			m_client.setTraceLogFile(boost::filesystem::path(traceLogFileNode.asString()));
```

Comment:
Looking at the test with `solc.log` passed as the parameter - will this only accept file names, or are fully qualified paths OK as well? If yes, what will happen if we pass an invalid path file here; or rather, if we do, boost will certainly complain, likely via exception (?), so is this handled appropriately?